content/attacks/posts/2023-06-06-Harmony’s Horizon.md

```diff
@@ -0,0 +1,33 @@
+﻿---
+date: 2022-06-23
+custodians: Harmony’s Horizon
+categories: Bridge Hack
+title: "Harmony's Horizon Bridge was the victim of a massive cyberattack from North Korea"
+---
+
+## Summary
+
+On Thursday, June 23, 2022, the Harmony Protocol team was notified of a malicious attack on their proprietary Horizon Bridge. Blockchain bridges, also known as cross-chain bridges, facilitate communication between different blockchain networks and allow users to transfer assets from one network to another. Using Harmony's Horizon Bridge, users can move assets, including tokens, stablecoins, and NFTs, between the Ethereum, Binance Smart Chain, and Harmony blockchains.
+At 5:30 AM Pacific Standard Time, several transactions compromised the bridge. The hackers were able to steal the following assets: Frax (FRAX), Wrapped Ether (WETH), Aave (AAVE), Sushi (SUSHI), Frax Share (FXS), AAG (AAG), Binance USD (BUSD), Dai (DAI), Tether (USDT), Wrapped BTC (WBTC), and USD Coin (USDC). Some of these tokens have been exchanged for ETH on decentralized exchanges.
+
+## Attackers
```

Comment:
Focus on the attackers rather than the events. Adding specific pieces of evidence that were used in attributing the attack to the North Koreans will be extremely valuable.

Reply:
the section still includes events instead of attackers' description and evidence

Reply:
I'm sorry, it's fixed